backend/typescript/seeders/20211114043430-seed-donations.ts

```diff
@@ -2,7 +2,6 @@ import { Seeder } from "../umzug";
 
 const seedUsers = [
   {
-    id: 1,
```

Comment:
I am just fixing the seeding in this PR cuz there was a change being made anyways. We should not be passing in id because for some reason postgres does not auto increment id when passed in explicitly. So when we create using api, it still tries to add with id 1 which results into a duplicate key error.